README.md

```diff
@@ -53,7 +55,7 @@ Note that if you change the port that the server is running on you'll need to ed
 ## Building the UI package (release mode)
 
 Simply run the build command in the `tracker` directory:
-```yarn build```
+```> yarn build```
```

Comment:
I generally prefer not including any leading marks in code blocks cause copying is harder.

Tiny point here, but for someone unfamiliar it's not always clear.

Reply:
It was the one place in the doc that it WASN'T doing that, so it was more a consistency thing than anything else.